libs/agora/services/.eslintrc.json

```diff
@@ -29,7 +29,8 @@
             "style": "kebab-case"
           }
         ],
-        "@angular-eslint/prefer-standalone": "off"
+        "@angular-eslint/prefer-standalone": "off",
+        "@angular-eslint/prefer-inject": "warn"
```

Comment:
I added this rule myself to turn the corresponding error into a warning. The projects that include this rule should be updated in the future to use Angular's `inject()` function.